controller/hybridgateway/converter/http_route.go

```diff
@@ -256,14 +400,24 @@ func (c *httpRouteConverter) translate(ctx context.Context, logger logr.Logger)
 				WithControlPlaneRef(*cp).
 				WithOwner(c.route).Build()
 			if err != nil {
-				// TODO: decide how to handle build errors in converter
-				// For now, skip this resource
+				log.Error(logger, err, "Failed to build KongService resource, skipping rule",
+					"service", serviceName,
+					"upstream", upstreamName)
+				translationErrors = append(translationErrors, fmt.Errorf("failed to build KongService %s: %w", serviceName, err))
 				continue
 			}
 			c.outputStore = append(c.outputStore, &service)
+			log.Debug(logger, "Successfully built KongService resource",
+				"service", serviceName)
 
 			// Build the kong route resource.
 			routeName := namegen.NewName(httpRouteName, cpRefName, utils.Hash32(rule.Matches)).String()
+			log.Debug(logger, "Building KongRoute resource",
```

Comment:
Building logs are at Trace level for Plugins and PluginBindings: we may want to either change the others to Trace or change Plugin and PluginBinding build logs to Debug.
```suggestion
			log.Trace(logger, "Building KongRoute resource",
```